app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -775,6 +774,14 @@ class StatePlayerRecyclerViewAssembler private constructor(
       }
     }
 
+    private val isCorrectAnswer: ObservableField<Boolean> = ObservableField(false)
+
+    /** @param isCorrectAnswer is restored to its updated value on configuration change */
+    fun isAnswerCorrect(isCorrectAnswer: Boolean): Builder {
```

Comment:
This isn't the correct way to be doing this--the builder is designed to set up high-level UI features, not change super specific state.

The underlying issue that we're trying to solve is retaining state across configuration changes. We probably shouldn't be using the view model below, either, since that's set up incorrectly today (view models shouldn't be retrieved using ViewModelProvider since they can leak fragments/activities). Sorry--this probably wasn't clear.

It seems like perhaps the clearest way to solve this is to have the assembler itself ensure isCorrectAnswer is properly set in compute() based on whether we're in a completed state & the answer submitted is correct. I think if you did that approach it might also simplify the overall solution.